metrics/metrics.go

```diff
@@ -112,6 +112,12 @@ func NewStatter(addr, prefix string) (Statter, error) {
 	return Statter{suffix, s}, nil
 }
 
+// NewNoopStatter returns a new statsd.NoopClient wrapper
+func NewNoopStatter() Statter {
```

Comment:
We already have a `NewNoopScope`, want to use that instead (and change the function signatures to use `Scope` instead of `Statter`?